src/components/Home.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import styles from '../CSS/Home.module.css';
+
+const Home = () => {
+  const { news, categories, totalResults } = useSelector((state) => state.news);
+  const [selectedItem, setSelectedItem] = useState(null);
+
+  const handleCardClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  return (
+    <div className={styles.homeCont} data-testid="home">
+      <div className={styles.headCont}>
+        <h1 className={styles.titleN}>News</h1>
+        <p>
+          Total Results Sum:
+          {' '}
+          {totalResults}
+        </p>
+      </div>
+      <div className={styles.homeCard}>
+        {news.map((item) => (
+          <div className={styles.card} key={item.title} data-testid="card">
+            <Link to={`/detail/${encodeURIComponent(item.title)}`} className={styles.single_card}>
+              <div
+                onClick={() => handleCardClick(item)}
+                onKeyDown={() => handleCardClick(item)}
+                role="link"
+                tabIndex={0}
+                className={styles.card_content}
+              >
+                <div className={styles.arrow_wrapper}>
+                  <span>
+                    <FaArrowAltCircleRight className={styles.arrow_icon} />
+                  </span>
+                </div>
+                <img
+                  src={item.image}
+                  alt={item.title}
+                  className={styles.card_image}
+                />
+                <p className={styles.titlecd}>{item.title}</p>
+                <p className={styles.titlecd}>
+                  Total Results:
+                  {' '}
+                  {categories[item.source.name]}
+                </p>
+              </div>
+            </Link>
+          </div>
+        ))}
+      </div>
+      {selectedItem && (
+        <div className={styles.selected_item}>
+          <img
+            src={selectedItem.urlToImage}
+            alt={selectedItem.title}
+            className={styles.selected_item_image}
+          />
+          <p>{selectedItem.title}</p>
+          <p>
+            Description:
+            {' '}
+            {selectedItem.description}
+          </p>
+          <p>
+            URL:
```

Comment:
- [ ] as part of the requirements you  should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
* Typography: font face, size, and weight
* Layout: composition and space between elements.


currently, your design missing some enhancements to match the guideline including : icon colors , space between elements , margins etc , kindly try to be close as much as you can with the design guidelines 🙏  


Your Design         | Design guidelines 
:-------------------------:|:-------------------------:
![image](https://github.com/Christelle-12/react-redux-capstone/assets/89282221/ce73f3b9-b44e-4819-b346-fd6bb675d3d3) | ![image](https://github.com/Christelle-12/react-redux-capstone/assets/89282221/5ccb0196-979c-42f8-af13-9661b37897dd)